client/src/charts/progressGauge.js

```diff
@@ -0,0 +1,40 @@
+import "./progressGauge.scss";
+import { successDarkColor } from "../core/color_defs";
+import { Format } from "../components";
+
+export default class ProgressGauge extends React.Component {
+  render() {
+    const { value, total_value, finished_color } = this.props;
+    const percentage = value / total_value;
+
+    const progress = _.times(total_value, (i) => {
+      if (i < value) {
+        return (
+          <div
```

Comment:
`div` is a block element : https://en.wikipedia.org/wiki/HTML_element#Block_elements
`span` is an inline element: https://en.wikipedia.org/wiki/HTML_element#Inline_elements

In this case, `span` would be more appropriate